app/Http/Controllers/PostController.php

```diff
@@ -13,7 +15,9 @@ class PostController extends Controller
      */
     public function index()
     {
-        //
+        $items = Post::orderBy('created_at', 'desc')->paginate(config('number.paginate'));
+
+        return view('index', compact('item'));
```

Comment:
sao ko sửa tên biến bên view luôn đi em, 
```suggestion
        return view('index', compact('items'));
```

Reply:
ok anh

Reply:
ok anh mà ko sửa à em?